src/lib/grid-list/tile-coordinator.ts

```diff
@@ -99,6 +101,8 @@ export class TileCoordinator {
       // If there are no more empty spaces in this row at all, move on to the next row.
       if (gapStartIndex == -1) {
         this._nextRow();
+        gapStartIndex = this.tracker.indexOf(0, this.columnIndex);
+        gapEndIndex = this._findGapEndIndex(gapStartIndex);
```

Comment:
I believe this also ins't going to do anything, as these will be rewritten after the `continue`

Reply:
once the code hit continue, it will skip the rest in the block and jump to the "while" check, without the new lines, the endIndex will remain as the last row.